history/taskana-simplehistory-provider/src/test/java/acceptance/query/QueryWorkbasketHistoryAccTest.java

```diff
@@ -277,6 +270,10 @@ void should_ReturnHistoryEvents_In_DifferentOrders() throws InvalidArgumentExcep
         historyService
             .createWorkbasketHistoryQuery()
             .orderByCustomAttribute(1, SortDirection.ASCENDING)
+            // values of custom1 do not differ and are larger than 3 (n=5).
+            // Therefore, in order to guarantee the sort order,
+            // we have to define a secondary (unique) sort option
+            .orderById(SortDirection.ASCENDING)
```

Comment:
Why donÄt simply change the values inside the test data instead of extending the official API?

Reply:
I could "simply" change the values inside the test data. Doing that many tests will fail.
Furthermore I believe the addition of `orderById` in general is a good thing, so that our users can determine a unique sort order.